app/src/main/java/com/example/habitsmasher/MainActivity.java

```diff
@@ -58,4 +59,41 @@ public boolean onSupportNavigateUp() {
         return NavigationUI.navigateUp(navController, appBarConfiguration)
                 || super.onSupportNavigateUp();
     }
-}
\ No newline at end of file
+
+    /**
+     * This method allows us to inflate the menu bar, where we can add items if the bar is present
+     * @param menu menu to inflate
+     * @return true
+     */
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        // Inflate the menu, adding items to the action bar if present
+        getMenuInflater().inflate(R.menu.menu, menu);
+        return true;
+    }
+
+    /**
+     * This method is called when an item is selected in the action bar
+     * @param item the item selected
+     * @return true if item selected, false otherwise
+     */
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        if (item.getItemId() == R.id.follow_user_search_button) {
+            openFollowUserDialog();
+            return true;
+        }
+
+        return super.onOptionsItemSelected(item);
+    }
+
+    /**
+     * This helper method is responsible for opening the FollowUserDialog box
+     */
+    private void openFollowUserDialog() {
+        FollowUserDialog followUserDialog = new FollowUserDialog();
+        followUserDialog.setCancelable(true);
+        followUserDialog.setTargetFragment(getSupportFragmentManager().getFragments().get(0), 1);
+        followUserDialog.show(getSupportFragmentManager(), "FollowUserDialog");
+    }
```

Comment:
These methods allow us to add a button to the menu bar up top, and open the `FollowUserDialog` if it's clicked. This is in the `MainActivity` because the user should be able to access this throughout the app.